README.md

```diff
@@ -12,7 +12,44 @@ We use [Nuxt Content](https://content.nuxtjs.org/) to render markdown content
 in a web page. Add `@nuxt/content` under modules in `.theme/nuxt.congfig.ts`
 
 Markdown files are placed at the root of to give focus on more to content than
-its theme layout.
+its theme layout. 
+
+### Conventions
+
+We are using our conventions when creating new or adding/editing content
+on existing markdown files. The aim of our conventions are keeping the markdown
+files consistent in itself. Links, images etc. should be working on any
+platform this markdown file is viewed such as github. Following are the
+conventions we are using, ones that require more explanation are explained in
+their own sections below.
```

Comment:
@alptuncs kısaltabilir misin, biraz laf kalabalığı var gibi. Our conventions, the conventions we use 3 kere yazmışız. Başlık zaten conventions ya. Daha pratik hale getirir misin

Reply:
Anlatımı bozmadan laf kalabalığını azaltmaya çalıştım.